lib/traject/macros/met_csv.rb

```diff
@@ -40,11 +40,14 @@ def generate_creator
     end
 
     def artist_date_range(row)
-      @artist_date_range ||= [row[BEGIN_DATE], row[END_DATE]].select(&:present?).join(' - ')
+      [row[BEGIN_DATE], row[END_DATE]].select(&:present?).join(' - ')
     end
 
     def artist_role_bio(row)
-      @artist_role_bio ||= "#{artist_date_range(row)} (#{[row[ROLE], row[BIO]].select(&:present?).join(' ; ')})"
+      role = [row[ROLE], row[BIO]].select(&:present?).join(' ; ')
+      artist = artist_date_range(row)
```

Comment:
is `artist` the right variable name? it seems like a date to me.

Reply:
On this line it is a date range, on the next line I add the role & bio to this variable.

Reply:
Sure, but it's never an artist name, is it? `artist_info`, maybe?

Reply:
Sure, names are hard.